airflow/providers/apache/hdfs/log/webhdfs_task_handler.py

```diff
@@ -0,0 +1,186 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+import os
+import pathlib
+from typing import Any
+
+from airflow.compat.functools import cached_property
+from airflow.configuration import conf
+from airflow.models import TaskInstance
+from airflow.providers.apache.hdfs.hooks.webhdfs import WebHDFSHook
+from airflow.utils.log.file_task_handler import FileTaskHandler
+from airflow.utils.log.logging_mixin import LoggingMixin
+
+
+class WebHDFSTaskHandler(FileTaskHandler, LoggingMixin):
+    """A TaskHandler that uploads logs to HDFS once done. During the run, it will show the executor logs."""
+
+    trigger_should_wrap = True
+
+    def __init__(
+        self,
+        base_log_folder: str,
+        webhdfs_log_folder: str,
+        filename_template: str | None = None,
+    ):
+        super().__init__(base_log_folder=base_log_folder, filename_template=filename_template)
+        # Overwrite the location to write task logs to.
```

Comment:
```suggestion
```

That comment makes it look like you mean "replace the file" i.e. delete any existing log in the remote path

but i don't think that's what you mean